clojure.lang.ExceptionInfo: :auto not supported on headerless data. {}

PigPen Version: 0.3.1-20150825.165235-4
java version: 1.7
hadoop version 2.4.0
java version: 1.7
pig versions: 0.12.0
AMI version:3.4.0
Hadoop distribution:Amazon 2.4.0

At the end of our map step we are seeing the following exception: 

Caused by: clojure.lang.ExceptionInfo: :auto not supported on headerless data. {}
	at clojure.core$ex_info.invoke(core.clj:4403)
	at taoensso.nippy$get_auto_compressor.invoke(nippy.clj:558)
	at taoensso.nippy$thaw$thaw_data__10665.invoke(nippy.clj:599)
	at taoensso.nippy$thaw.doInvoke(nippy.clj:644)
	at clojure.lang.RestFn.invoke(RestFn.java:410)
	at pigpen.pig.runtime$fn__12071.invoke(runtime.clj:108)
	at pigpen.runtime$fn__2746$G__2741__2751.invoke(runtime.clj:186)
	at clojure.core$mapv$fn__6311.invoke(core.clj:6353)
	at clojure.core.protocols$fn__6074.invoke(protocols.clj:79)
	at clojure.core.protocols$fn__6031$G__6026__6044.invoke(protocols.clj:13)
	at clojure.core$reduce.invoke(core.clj:6289)
	at clojure.core$mapv.invoke(core.clj:6353)
	at pigpen.pig.runtime$fn__12069.invoke(runtime.clj:111)
	at pigpen.runtime$fn__2746$G__2741__2751.invoke(runtime.clj:186)
	at clojure.core$map$fn__4245.invoke(core.clj:2559)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:484)
	at clojure.core$seq.invoke(core.clj:133)
	at clojure.core$apply.invoke(core.clj:624)
	at clojure.core$mapcat.doInvoke(core.clj:2586)
	at clojure.lang.RestFn.invoke(RestFn.java:423)
	at pigpen.pig.runtime$fn__12067.invoke(runtime.clj:115)
	at pigpen.runtime$fn__2746$G__2741__2751.invoke(runtime.clj:186)
	at pigpen.pig.runtime$exec_initial.invoke(runtime.clj:295)
	at pigpen.pig.runtime$udf_algebraic.invoke(runtime.clj:337)
	at clojure.lang.Var.invoke(Var.java:388)
	at pigpen.PigPenFnAlgebraic$Initial.exec(PigPenFnAlgebraic.java:104)
	at pigpen.PigPenFnAlgebraic$Initial.exec(PigPenFnAlgebraic.java:88)
	at org.apache.pig.backend.hadoop.executionengine.physicalLayer.expressionOperators.POUserFunc.getNext(POUserFunc.java:330)

The map phase get to %100 and all the inputs seem to be processed.

Could it be some compression library thats not installed?

Any ideas where to look?

Thanks 
-Bob

--- a/pigpen-core/build.gradle
+++ b/pigpen-core/build.gradle
@@ -3,7 +3,7 @@ dependencies {
     compile 'org.clojure:clojure:1.6.0'
     compile 'org.clojure:data.json:0.2.5'
     compile 'org.clojure:data.csv:0.1.2'
-    compile 'com.taoensso:nippy:2.9.1'
+    compile 'com.taoensso:nippy:2.10.0'
     compile 'prismatic:schema:0.3.3'
 }
 

--- a/pigpen-pig/src/main/clojure/pigpen/pig/runtime.clj
+++ b/pigpen-pig/src/main/clojure/pigpen/pig/runtime.clj
@@ -61,7 +61,7 @@ possible as it's used at runtime."
     (.add tuple)))
 
 (defn ^:private pig-freeze [value]
-  (DataByteArray. (freeze value)))
+  (DataByteArray. (freeze value {:compressor nil, :skip-header? true})))
 
 (defn ^:private pig-freeze-with-nils [value]
   (if-not (nil? value)
@@ -105,7 +105,9 @@ possible as it's used at runtime."
 (extend-protocol HybridToClojure
   DataByteArray
   (rt/hybrid->clojure [^DataByteArray value]
-    (-> value (.get) thaw))
+    (-> value (.get) (thaw {:compressor nil
+                            :encryptor  nil
+                            :v1-compatibility? false})))
   Tuple
   (rt/hybrid->clojure [^Tuple value]
     (->> value (.getAll) (mapv rt/hybrid->clojure)))

